geopy/geocoders/base.py

```diff
@@ -160,14 +160,14 @@ def _call_geocoder(
                     raise GeocoderServiceError(message)
             elif isinstance(error, URLError):
                 if "timed out" in message:
-                    raise GeocoderTimedOut('Service timed out')
+                    raise GeocoderTimedOut('URLError: Service timed out')
                 elif "unreachable" in message:
                     raise GeocoderUnavailable('Service not available')
             elif isinstance(error, SocketTimeout):
-                raise GeocoderTimedOut('Service timed out')
+                raise GeocoderTimedOut('Socket Service timed out')
             elif isinstance(error, SSLError):
                 if "timed out" in message:
-                    raise GeocoderTimedOut('Service timed out')
+                    raise GeocoderTimedOut('SSLError Service timed out')
```

Comment:
fwiw SSLErrors are almost never triggerd by a timeout, it's usually a cert mismatch or a handshake error. `SSLError connection error` would be more accurate